api/database.go

```diff
@@ -37,6 +37,60 @@ func (e SyncStatus) String() string {
 	return ""
 }
 
+// DatabaseRaw is the store model for an Database.
+// Fields have exactly the same meanings as Database.
+type DatabaseRaw struct {
+	ID int
+
+	// Standard fields
+	CreatorID int
+	CreatedTs int64
+	UpdaterID int
+	UpdatedTs int64
+
+	// Related fields
+	ProjectID      int
+	InstanceID     int
+	SourceBackupID int
+
+	// Domain specific fields
+	Name                 string
+	CharacterSet         string
+	Collation            string
+	SchemaVersion        string
+	SyncStatus           SyncStatus
+	LastSuccessfulSyncTs int64
+	Labels               string
```

Comment:
I don't think Labels is part of DatabaseRaw as we see it's not read from store.

Reply:
It's used here: https://github.com/dragonly/bytebase/blob/dragonly/refactor/data-model-database/server/issue.go#L1116

I think we can add a TODO here, and change to use composed `Database` in the next refactoing process (spliting packages).

Reply:
It's composed as part of database, but it shouldn't be part of databaseRaw.

Reply:
Oops, seems like a leftover, should've changed it to database.

Reply:
Fixed, PTAL.